src/transformation/to_trimesh/cuboid_to_trimesh.rs

```diff
@@ -3,19 +3,14 @@ use crate::math::{Point, Real};
 use crate::shape::Cuboid;
 use crate::transformation::utils;
 
-#[cfg(feature = "dim3")]
-use na::Point3 as PointDim;
-#[cfg(feature = "dim3")]
-use na::{self, Point3};
-
 #[cfg(feature = "dim2")]
 use na::Point2 as PointDim;
 #[cfg(feature = "dim2")]
 use na::{self, Point2};
```

Comment:
Are these still needed?

Reply:
This was addressed in a previous PR :ok_hand: